go/ephemeral/lib.go

```diff
@@ -57,7 +57,7 @@ func (e *EKLib) checkLoginAndPUK(ctx context.Context) error {
 // to fully release it.
 func (e *EKLib) ShouldRun(ctx context.Context) bool {
 	g := e.G()
-	willRun := g.Env.GetFeatureFlags().UseEphemeral() || g.Env.GetRunMode() == libkb.DevelRunMode || g.Env.RunningInCI()
+	willRun := g.Env.GetFeatureFlags().Admin() || g.Env.GetRunMode() == libkb.DevelRunMode || g.Env.RunningInCI()
```

Comment:
this will make keygen happen for anyone with the `admin flag

Reply:
This part of the PR affects prod clients, right? Not just dev? Maybe we should make it a separate one-line commit, in case we need to revert it?

Reply:
good idea, i'll back this out and flip it in a subsequent PR